lib/Frontend/Frontend.cpp

```diff
@@ -456,7 +456,7 @@ bool CompilerInstance::setupCASIfNeeded(ArrayRef<const char *> Args) {
   const auto &Opts = getInvocation().getCASOptions();
   auto MaybeDB= Opts.CASOpts.getOrCreateDatabases();
   if (!MaybeDB) {
-    Diagnostics.diagnose(SourceLoc(), diag::error_cas,
+    Diagnostics.diagnose(SourceLoc(), diag::error_cas_setup,
```

Comment:
> error: CAS setup error: <error>

Maybe we could have a more specific error like

> error: cannot open CAS database: <error>

or if we want to look more like the clang error:

> error: CAS cannot be initialized from the specified '-cas-*' options: <error>